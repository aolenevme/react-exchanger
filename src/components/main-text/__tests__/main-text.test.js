import React from "react";
import renderer from "react-test-renderer";

import MainText from "../main-text.js";

describe("<MainText />", () => {
    it("is rendered correctly", () => {
        expect(renderer
            .create(<MainText>GBP</MainText>)
            .toJSON()).toMatchSnapshot();
    });
});
