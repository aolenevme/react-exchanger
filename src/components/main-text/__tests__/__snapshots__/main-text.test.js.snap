// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainText /> is rendered correctly 1`] = `
.c0 {
  font-size: 3rem;
  line-height: 3rem;
  color: #f3f6ff;
}

<span
  className="c0"
>
  GBP
</span>
`;
